packages/app/src/components/PageComment/CommentEditor.jsx

```diff
@@ -417,6 +417,7 @@ CommentEditor.propTypes = {
   commentCreator: PropTypes.string,
   onCancelButtonClicked: PropTypes.func,
   onCommentButtonClicked: PropTypes.func,
+  isComment:  PropTypes.bool,
```

Comment:
Do you need this?

Reply:
These props is used to distinguish if the editor is initialized from page editor or comment editor since they both use the same components

Reply:
hmm..
I think the line 314 is setting `isComment={true}` regardless the line 420 exists or not.

Reply:
i had set `isComment={true}` before, but the eslint plugin seems to change it to `isComment` by default, so I set `isComment` to state as true